### PR TITLE
Added detectConnectedDevices fields to v1 Flow API

### PR DESCRIPTION
**Pull request by niksv**
Part of #2582

### CODE REVIEW

**Comment by rtretyak** [comment]
I don't think it's a good idea to enable this in public API until its working.
Second point: because of enabled v1->v2 mapping for all flow create requests in prod we can't add this feature only to v1.

File: services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Flow.java [hunk 1]
```diff
@@ -85,12 +85,24 @@
     @Property(name = "src_vlan")
     private int srcVlan;
 
+    @Property(name = "detect_src_lldp_connected_devices")
+    private boolean detectSrcLldpConnectedDevices;
```
Comment (sergii-iakovenko) [inline services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Flow.java]: Please, consider wrapping the new properties into a dedicated class. This class will be mapped to a set of Flow fields in the storage.

Comment (niksv) [reply services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Flow.java]: added converter

File: services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Flow.java [hunk 2]
```diff
@@ -493,6 +499,17 @@ private FlowPathStatus getFlowPrioritizedPathStatus(FlowPath... flowPaths) {
                 .orElse(null);
     }
 
+    /**
+     * Return detect connected devices flags.
+     */
+    public void setDetectConnectedDevices(DetectConnectedDevices detectConnectedDevices) {
+        if (detectConnectedDevices == null) {
```
Comment (nikitamarchenko) [inline services/src/kilda-core/kilda-model/src/main/java/org/openkilda/model/Flow.java]: maybe use Optional here

**Pull request merged**